pkg/db/db_init.go

```diff
@@ -54,21 +54,22 @@ func (d *dbConn) DBInit() {
 		log.Fatalf("Error creating workers table: %v", err)
 	}
 
-	_, err = db.Exec("CREATE TABLE IF NOT EXISTS worker_metrics" +
-		"(worker_id CHAR(16) NOT NULL, " +
-		"id INT AUTO_INCREMENT PRIMARY KEY, " +
-		"time DATETIME(6), " +
-		"name VARCHAR(255), " +
-		"value TEXT, " +
-		"is_objective TINYINT)")
+	_, err = db.Exec(`CREATE TABLE IF NOT EXISTS worker_metrics
+		(worker_id CHAR(16) NOT NULL,
+		id INT AUTO_INCREMENT PRIMARY KEY,
+		time DATETIME(6),
+		name VARCHAR(255),
+		value TEXT,
+		is_objective TINYINT,
+		FOREIGN KEY (worker_id) REFERENCES workers(id) ON DELETE CASCADE)`)
```

Comment:
Could you replace `" +` to ` on other tables?

Reply:
My idea was to minimize the size of diff. It might be better to tidy all the things now. Just a sec.